mtags/src/main/scala-2/scala/meta/internal/pc/Identifier.scala

```diff
@@ -62,7 +62,7 @@ object Identifier {
   def backtickWrap(s: String): String = {
     if (s.isEmpty) "``"
     else if (s(0) == '`' && s.last == '`') s
-    else if (needsBacktick(s)) '`' + s + '`'
+    else if (needsBacktick(s)) "" + ('`') + s + '`'
```

Comment:
I've seen this in a couple places where it prefaces with an empty string. What's the reasoning for that?

Reply:
I was also confused, but it seems the reason is https://scalacenter.github.io/scala-3-migration-guide/docs/incompatibilities/dropped-features.html#any2stringadd-conversion

Reply:
huh, alright, thanks for linking that. I hope this doesn't become confusing for someone in the future that comes across this and wonders why there is a `""` prefacing it. I can see myself seeing that just removing it 😆

Reply:
Hm. I've just realized that this line might look less weird if at least the first backtick was a type of `String`:
```scala
else if (needsBacktick(s)) "`"+ s + "`"`
```

Reply:
ach yes, I didn't notice the comments when merging, sorry!